src/App.jsx

```diff
@@ -2,18 +2,47 @@ import { useState } from 'react'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
   const [name, setName] = useState("");
-  const [day, setDay] = useState(0)
+  const [day, setDay] = useState(1)
   const [month, setMonth] = useState('')
+  const [daysToGo, setDaysToGo] = useState(0)
+  const [isNotSubmitted, setIsNotSubmitted] = useState(true)
 
+  const dayMin = 1
+  const dayMax = 31
 
+  const today = new Date()
+  let birthday;
+
+  const updateDay = event => {
+    const birthdayDay = Math.max(dayMin, Math.min(dayMax, Number(event.target.value)))
+    setDay(birthdayDay)
+  }
+
+  const submit = () => {
+    setIsNotSubmitted(false)
+    birthday = new Date(`${month.toString()}-${day.toString()}`)
```

Comment:
I think it would be nice to do the processing of the birthday in a separate helper file so that you keep this file to just dealing with the actual visual components